data/json/itemgroups/mansion.json

```diff
@@ -400,7 +419,7 @@
       [ "pike_fake", 50 ],
       { "group": "broadsword_fake+scabbard", "prob": 17 },
       [ "mace_fake", 6 ],
-      [ "morningstar_fake", 5 ]
+      [ "morningstar_fake", 5 ],
```

Comment:
```suggestion
      [ "morningstar_fake", 5 ]
```

Reply:
Shoo, extra seperator, shoo!

Reply:
Nope, thx. Just moving fast and breaking my own things.